### DISCUSSION TITLE
Jarring effect from state change when lights turn on

### DISCUSSION THREAD

**Discussion by Nsalty84** [basnijholt/adaptive-lighting#298] [Q&A]
I love adaptive lighting. Have mine dialled in nicely and the changes over the course of the day you don't even notice. One thing I haven't figured out yet is how to avoid the jarring effect when a light hasn't been on and its previous state is vastly different from the current adaptive lighting state. This results in the light turning on in the previous state before adaptive lighting kicks in and set it to what it should be. Even with the transitions set to 0. Wondering if anyone else has the same problem?

**Comment by daFreeMan** [comment]
I would think that the `initial_transition` setting would be the one you'd want. I have mine set to 30 (seconds) and when they come on (occupancy trigger) they smoothly ramp up from very dim to "whatever is appropriate now".
